python/ray/_private/log_monitor.py

```diff
@@ -508,7 +508,7 @@ def is_proc_alive(pid):
         try:
             os.kill(pid, 0)
             return True
-        except OSError:
+        except OSError, ProcessLookupError:
```

Comment:
any other errors we should expect as well?

Reply:
hmm. https://docs.python.org/3/library/exceptions.html#OSError looks ProcessLookupError is already subclass of OSError?